data/interface.go

```diff
@@ -107,6 +107,35 @@ type TransactionHandler interface {
 	SetSndAddr([]byte)
 }
 
+// LogHandler defines the type for a log resulted from executing a transaction or smart contract call
+type LogHandler interface {
+	IsInterfaceNil() bool
+
+	// GetAddress returns the address of the sc that was originally called by the user
+	GetAddress() []byte
+	GetLogEvents() []EventHandler
```

Comment:
Since the methods on the interfaces in this file did not have any comments, I only added on those worth explaining the difference. But you are right, I added comments on all of them to have a consistency